stable/redis-ha/README.md

```diff
@@ -58,28 +58,28 @@ $ helm install \
-| `servers.serviceType`            | Set to "LoadBalancer" to enable access from the VPC                                                                          | ClusterIP                                                 |
-| `servers.annotations`            | See Appliance mode                                                                                                           | ``                                                        |
-| `rbac.create`                    |  whether RBAC resources should be created                                                                                    | true                                                      |
-| `serviceAccount.create`          | whether a new service account name that the agent will use should be created.                                                | true                                                      |
-| `serviceAccount.name`            | service account to be used.  If not set and serviceAccount.create is `true` a name is generated using the fullname template. | ``                                                        |
+| Parameter                        | Description                                           | Default                                                   |
+| -------------------------------- | ----------------------------------------------------- | --------------------------------------------------------- |
+| `redis_image`                    | Redis image                                           | `quay.io/smile/redis:4.0.6r2`                             |
+| `resources.server`               | CPU/Memory for redis nodes resource requests/limits  | Memory: `200Mi`, CPU: `100m`                              |
+| `resources.sentinel`             | CPU/Memory for sentinel node resource requests/limits | Memory: `200Mi`, CPU: `100m`                              |
+| `replicas.servers`               | Number of redis master/slave pods                     | 3                                                         |
+| `replicas.sentinels`             | Number of sentinel pods                               | 3                                                         |
+| `nodeSelector`                   | Node labels for pod assignment                        | {}                                                        |
+| `tolerations`                    | Toleration labels for pod assignment                  | []                                                        |
+| `servers.serviceType`            | Set to "LoadBalancer" to enable access from the VPC   | ClusterIP                                                 |
+| `servers.annotations`            | See Appliance mode                                    | ``                                                        |
+| `serviceAccount.create`                    | If true, create & use RBAC resources                  | `false`                                                   |
```

Comment:
If the goal is to keep the table aligned, `serviceAccount.create` is out of line.